src/SFML/Graphics/View.cpp

```diff
@@ -47,14 +47,13 @@ m_invTransformUpdated(false)
 
 ////////////////////////////////////////////////////////////
 View::View(const FloatRect& rectangle) :
-m_center(),
-m_size(),
+m_center(rectangle.getCenter()),
+m_size(rectangle.size),
 m_rotation(),
 m_viewport({0, 0}, {1, 1}),
 m_transformUpdated(false),
 m_invTransformUpdated(false)
 {
-    reset(rectangle);
```

Comment:
Please revert this back to its original state unless these changes are necessary.

Reply:
Yeah, this one is not!